Разработка сайта портфолио репетитора

makaroveo opened the pull request.

makaroveo committed:
Основные разделы, react router

Добавил основные разделы, основную структуру, добавил многостраничность через React Router

makaroveo committed:
Пересборка идеи

Пришла идея сделать свое портфолио, как репетитора математики, так как это текущая проф деятельность.
Сделаны структуры страниц Главная, О занятиях, Контакты.
Начало разработки Header

makaroveo committed:
Header

Прототип header, начало создания aboutMe

makaroveo committed:
Сделал раздел «Обо мне»

Доделал раздел обо мне, возможно, позже, будут доработаны media запросы

makaroveo committed:
Отредактирован раздел услуги

Сделан раздел услуги, добавлены медиа запросы, изменение стиля при наведении

makaroveo committed:
Раздел отзывов

Сделал каруcель отзывов, медиа запросы, использовал библиотеку React slick

makaroveo committed:
Footer

Создание футера

makaroveo committed:
Добавлен раздел О формате на страницу «О занятих»

Сделал раздел «О формате», реализовал плавное появление блоков с преимуществами

makaroveo committed:
Раздел инструменты

Здесь довольно долго решал вопрос с медиа запросами, они проработоны довольно детально.

Также реализована задача «активного» сервиса

makaroveo committed:
Контакты

Раздел контакты реализован

makaroveo committed:
Блок контаков

Реализовал блок контактов, указал минимальлную высоту для более красивого вида

makaroveo committed:
dependencies devdependencies

dependencies devdependencies изменил, очистил readme

makaroveo committed:
Добавил лого, очистил Readme

makaroveo committed:
Update README.md

makaroveo committed:
Исправил index.html

Исправил index.html, чтобы работали media на устройствах

makaroveo committed:
Merge branch 'dev2' of https://github.com/semonmakarov/react-portfolio into dev2

stepancar reviewed (changes requested):
Пожалуйста, сделайте по заданию. PR из форка не допускается.
Тогда я смогу приступить к ревью

makaroveo committed:
Структуризация стилей в App.css

makaroveo committed:
Разделение App.css на общие стили и специфичные

makaroveo committed:
eslint

makaroveo committed:
Контакты/ e-mail

А также начало работы с css modules

makaroveo committed:
Хочу попробовать реализовать через webpack.config.dev.js

makaroveo committed:
CSS modules всего проекта

Пересмотрел все файлы проекта, изменил название классов, используя camelCase нотацию для более удобного использования классов в js

makaroveo committed:
eslint

makaroveo committed:
Create .eslintrc.yml

makaroveo committed:
eslint

makaroveo committed:
@microsoft/eslint-formatter-sarif

makaroveo committed:
Workflow add

makaroveo committed:
fix workflow

makaroveo committed:
fix workflow and editorcongig
